providers/base/units/camera/jobs.pxu

```diff
@@ -119,13 +119,31 @@ estimated_duration: 20s
 depends: camera/detect
 requires: lsb.release >= '22.04'
 command:
-  timeout 120 camera_quality_test.py -d {{ name }} ||
-  (>&2 echo "Timeout computing score"; false)
+  camera_quality_test.py -d {{ name }}
 _description:
   Takes a picture and computes the quality based on a No-Reference image
   quality assessment algorithm called BRISQUE. This test will timeout and fail
   if the quality has not been compute within 120 seconds.
 
+unit: template
+template-resource: device
+template-filter: device.category == 'CAPTURE' and device.name != ''
+template-unit: job
+plugin: attachment
+template-engine: jinja2
```

Comment:
Any reason for using `jinja2` templating instead of the default?

Reply:
I was just using the same as the one we are using with the camera test. I am not sure about the differences between both.